src/vcpkg/commands.cpp

```diff
@@ -85,6 +86,7 @@ namespace vcpkg::Commands
         {"create", Create::perform_and_exit},
         {"deactivate", command_deactivate_and_exit},
         {"edit", Edit::perform_and_exit},
+        {"z-extract", extract_command_and_exit},
```

Comment:
I think this should be `command_extract_and_exit` or in a namespace `Extract::` to keep the same style as the other functions.